ISS/views/private_messages.py

```diff
@@ -152,3 +154,31 @@ def POST(self, request):
 
             return render(request, 'private_messages/compose.html', ctx)
 
+class PrivateMessageActions(utils.MethodSplitView):
+    staff_required = False
+    unbanned_required = True
```

Comment:
Add a `require_login = True` (like in `UnsubscribeFromThread` view in views/forum.py for example)  so that unauthenticated "users" can't submit this form. They woudldn't be able to get past the check on the message inbox later on but better to catch it before entering the view and opening a transaction.